techniques/general/G172.html

```diff
@@ -51,84 +49,77 @@
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、両端揃えをしていないバージョンのページを提供することである。</p>
+      <p>コンテンツ制作者は、レイアウトの目的でテキストを両端揃えにしたいと思う場合があるかもしれない。この場合、テキストの両端揃えを解除する機能を提供すればよい。そのためのコントロールは、探し出しやすく、アクセスしやすいように、ページの先頭の近くにおくとよい。</p>
       <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-         <p>This technique can be used in combination with a style switching technique to present a page that is a
-              <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-conforming-alternate-versions" target="terms">conforming alternate version</a>
-              for non-conforming content. Refer to
-              <a href="../css/C29">C29: Using a style switcher to provide a conforming alternate version</a>
-              and
-              <a href="https://w3c.github.io/wcag/understanding/conformance#conforming-alt-versions">Understanding Conforming Alternate Versions</a>
-              for more information.</p>
+         <p>このテクニックは、不適合コンテンツのための<a href="https://waic.jp/translations/WCAG22/#dfn-conforming-alternate-versions" target="terms">適合している代替版</a>のページを表示させるためのスタイル切り替えのテクニックとの組み合わせで用いることができる。詳しくは、<a href="../css/C29">C29: 適合している代替版を提供するために、スタイルスイッチャーを使用する</a>及び<a href="https://waic.jp/translations/WCAG22/Understanding/conformance#conforming-alt-versions" target="terms">適合している代替版を理解する</a>を参照。</p>
       </div>
 			</div>
-   </section><section id="examples"><h2>Examples</h2>
-      <section class="example" id="example-1"><h3>Example 1</h3>
+   </section><section id="examples"><h2>事例</h2>
+      <section class="example" id="example-1"><h3>事例 1</h3>
          
-            <p>A classic novel online is on a site that attempts to duplicate the look of the originally published work, which includes full justification. A button is provided near the top of the page saying "remove full justification" and a style switching technique is used to swap out the style sheet. The new style sheet aligns the text only on the left.</p>
+            <p>両端揃えで、元々出版された作業の外観の再現を試みたオンラインの古典小説がサイトにある。「両端揃えを解除する」と書かれたボタンがページの先頭近くにあり、スタイルシートを切り替えるためにスタイルを切り換える達成方法が使用されている。新しいスタイルシートでは、テキストを左揃えで配置する。</p>
          
       </section>
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../css/C19">C19: Specifying alignment either to the left or right in CSS</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Open a page with full justification.</li>
-            <li>Check that there is a feature to remove the full justification.</li>
-            <li>Check that the feature removes the full justification.</li>
+            <li>両端揃えのページを開く。</li>
+            <li>両端揃えを解除する機能があることを確認する。</li>
```

Comment:
check that ... の訳を、他のテクニックと合わせました。
```suggestion
            <li>両端揃えを解除する機能があることをチェックする。</li>
```